### DISCUSSION TITLE
buildAssetsDir doesn't overwrite default value

### DISCUSSION THREAD

**Discussion by engenes** [nuxt/nuxt#16693] [Questions] [answered]
I set `buildAssetsDir: '/_app/'` in nuxt.config.ts.

After build I got the default path in preview mode
![image](https://user-images.githubusercontent.com/38880660/193614359-691d9855-e080-4ed5-aec4-4f96103ef67f.png)

**Comment by TheAlexLichter** [comment, marked as answer]
Works fine for me [in this StackBlitz](https://stackblitz.com/edit/https-github-comnuxtframeworkdiscussions7970?file=nuxt.config.ts,package.json).

It is `app.buildAssetsDir` though.

<img width="319" alt="image" src="https://user-images.githubusercontent.com/640208/193620796-317d495e-94df-423f-ab60-595ab9396cbe.png">

**Reply by engenes** [reply]
My mistake.. Tnx🙏
